audit/k8s-gcr-audit-test-prod/iam.json

```diff
@@ -8,16 +14,79 @@
     },
     {
       "members": [
+        "serviceAccount:375340694213-compute@developer.gserviceaccount.com",
+        "serviceAccount:375340694213@cloudservices.gserviceaccount.com",
         "serviceAccount:service-375340694213@containerregistry.iam.gserviceaccount.com"
       ],
       "role": "roles/editor"
     },
     {
       "members": [
+        "serviceAccount:k8s-infra-gcr-promoter@k8s-gcr-audit-test-prod.iam.gserviceaccount.com"
+      ],
+      "role": "roles/errorreporting.admin"
```

Comment:
Well this is unexpected ("admin"?) ... but I guess we can review the actual permissions separately!

Reply:
This is in the test-prod project, which is covered by a special-case.